addons/base_import/static/src/xml/base_import.xml

```diff
@@ -173,55 +118,196 @@
         </div>
         <pre t-if="preview"><t t-esc="preview"/></pre>
     </t>
-    <ul t-name="ImportView.error">
-        <li t-foreach="errors" t-as="error"
-            t-attf-class="oe_import_report alert alert-#{error_value[0].type}">
-            <t t-if="error_value.length gt 1">
-                <t t-call="ImportView.error.multi.header">
-                    <t t-set="error" t-value="error_value[0]"/>
-                    <t t-set="error_length" t-value="error_value.length"/>
-                </t>
-                <ul>
-                    <t t-foreach="error_value.length" t-as="index">
-                        <li t-att-class="index gt 4 ? 'oe_import_report_more':''">
-                            <t t-call="ImportView.error.multi.body">
-                                <t t-set="error" t-value="error_value[index]"/>
-                                <t t-set="index" t-value="index"/>
-                            </t>
-                        </li>
-                        <li t-if="error_value.length gt 5 and index == 4" style="display: block;">
-                            <a href="#" class="oe_import_report_count">
-                                <t t-esc="more(error_value.length - 5)"/>
-                            </a>
-                        </li>
-                    </t>
-                </ul>
-                <t t-if="error_value[0].moreinfo" t-raw="info(error_value[0].moreinfo)"/>
 
-            </t>
-            <t t-else="">
-                <t t-call="ImportView.error.single">
-                    <t t-set="error" t-value="error_value[0]"/>
+    <t t-name="ImportView.error">
+        <div t-foreach="errors.length" t-as="index" t-attf-class="oe_import_report alert alert-#{errors[index].type}">
+            <span class="oe_import_report_message">
+                <b><t t-esc="errors[index].message"/></b>
+                <t t-if="errors[index].rows" t-call="ImportView.error.at">
+                    <t t-set="rows" t-value="errors[index].rows"/>
                 </t>
+            </span>
+            <t t-if="errors[index].moreinfo" t-call="ImportView.error.more_info">
+                <t t-set="messages" t-value="errors[index].moreinfo"/>
             </t>
-        </li>
-    </ul>
-    <t t-name="ImportView.error.multi.header">
-        <span class="oe_import_report_message">
-            <t t-esc="error.message"/>
-            <t t-esc="at_multi_header(error_length)"/>
-        </span>
-    </t>
-    <t t-name="ImportView.error.multi.body">
-        <span class="oe_import_report_message">
-            <t t-esc="at_multi(error.rows)"/>
-        </span>
+        </div>
     </t>
+
+    <div t-name="ImportView.fieldError" t-attf-class="oe_import_report alert alert-#{errors[0].type}">
```

Comment:
Sometimes "ImportView.fieldError", sometimes "ImportView.error.single", does not seem very consistent.